### PR TITLE
python3-curtsies: update to 0.4.1.

### PR DESCRIPTION
**Pull request by icp1994**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture: **x86_64**

### CODE REVIEW

File: srcpkgs/python3-curtsies/template [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3-blessed python3-cwcwidth"
+checkdepends="${depends} python3-pyte python3-pytest"
 short_desc="Curses-like terminal wrapper, with colored strings (Python3)"
 maintainer="Maurizio Porrato <maurizio.porrato@gmail.com>"
 license="MIT"
-homepage="https://github.com/thomasballinger/curtsies"
+homepage="https://pypi.org/project/curtsies/"
```
Comment (ahesford) [inline srcpkgs/python3-curtsies/template:12]:
```suggestion
homepage="https://github.com/bpython/curtsies"
```

File: srcpkgs/python3-cwcwidth/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-Cython python3-setuptools python3-wheel"
+makedepends="python3-devel"
+depends="python3"
+checkdepends="python3-pytest"
+short_desc="Python bindings for wc(s)width"
+maintainer="icp <pangolin@vivaldi.net>"
+license="MIT"
+homepage="https://pypi.org/project/cwcwidth/"
```
Comment (ahesford) [inline srcpkgs/python3-cwcwidth/template:13]:
```suggestion
homepage="https://github.com/sebastinas/cwcwidth"
```

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/python3-curtsies/template (hunk 1 above)
Comment (icp1994) [reply srcpkgs/python3-curtsies/template:12]: Yeah I should probably stop doing that. My reasoning was [this](https://github.com/void-linux/void-packages/pull/41386#discussion_r1059905844) btw.

Comment (ahesford) [reply srcpkgs/python3-curtsies/template:12]: The rule of thumb is that the homepage should be whatever upstream considers to be the official homepage. If PyPI provides a link to the project homepage, use the target of that link. If the project page is a GitHub repo and the GitHub repo lists another link in the project description, use the target of that link instead (even if PyPI takes you to the GH page).

**Pull request closed without merging**